### PR TITLE
Update client documentation

### PR DESCRIPTION
**Pull request by mikewalch**

### CODE REVIEW

File: _docs-2/getting-started/clients.md [hunk 1]
```diff
@@ -224,21 +215,37 @@ to efficiently return ranges of consecutive keys and their associated values.
 
 ### Scanner
 
-To retrieve data, Clients use a [Scanner], which acts like an Iterator over
-keys and values. Scanners can be configured to start and stop at particular keys, and
+To retrieve data, create a [Scanner] using [AccumuloClient]. A Scanner acts like an Iterator over
+keys and values in the table.
+
+If a [Scanner] is created without [Authorizations], it use all [Authorizations] granted
```
Comment (milleruntime) [inline _docs-2/getting-started/clients.md]: Typo "it use all"

File: _docs-2/getting-started/clients.md [hunk 2]
```diff
@@ -224,21 +215,37 @@ to efficiently return ranges of consecutive keys and their associated values.
...
+```java
+Scanner s = client.createScanner("table", new Authorizations("public"));
+```
+
+Scanners can be configured to start and stop at particular keys, and
 to return a subset of the columns available.
 
 ```java
-// specify which visibilities we are allowed to see
+// specify which visibilities should be returned
```
Comment (milleruntime) [inline _docs-2/getting-started/clients.md]: This comment is not very clear or accurate.  Perhaps something like "return data with visibilities that match specified auths"

Comment (mikewalch) [reply _docs-2/getting-started/clients.md]: Fixed now

File: _docs-2/getting-started/clients.md (hunk 1 above)
Comment (mikewalch) [reply _docs-2/getting-started/clients.md]: Fixed

**Pull request merged**